src/test/nxt_http_parse_test.c

```diff
@@ -762,15 +762,14 @@ nxt_http_parse_test_request_line(nxt_http_request_parse_t *rp,
         return NXT_ERROR;
     }
 
-    if (rp->complex_target != (test->complex_target | test->quoted_target)) {
+    if (rp->complex_target != test->complex_target) {
```

Comment:
Not that it matters now, but the previous check looks suspect.

AIUI `->complex_target` is supposed to be treated as a boolean value so checking it against `->complex_target | ->quoted_target` looks wrong, although it happens to work (assuming the two values are either 0 or 1)...

Reply:
Yep, it seems a tricky.

Reply:
Btw, it looks like the CI test doesn't run with `./build/tests`. But BB did.